sections/header.liquid

```diff
@@ -830,17 +851,51 @@
       "info": "t:sections.header.settings.menu_type_desktop.info"
     },
     {
-      "type": "checkbox",
-      "id": "show_line_separator",
-      "default": true,
-      "label": "t:sections.header.settings.show_line_separator.label"
+      "type": "select",
+      "id": "sticky_header_type",
+      "options": [
+        {
+          "value": "none",
+          "label": "t:sections.header.settings.sticky_header_type.options__1.label"
+        },
+        {
+          "value": "scroll-up",
+          "label": "t:sections.header.settings.sticky_header_type.options__2.label"
+        },
+        {
+          "value": "always",
+          "label": "t:sections.header.settings.sticky_header_type.options__3.label"
+        }
+      ],
+      "default": "none",
+      "label": "t:sections.header.settings.sticky_header_type.label"
+    },
+    {
+      "type": "select",
+      "id": "reduce_logo_scroll",
+      "options": [
+        {
+          "value": "reduce-0",
+          "label": "t:sections.header.settings.reduce_logo_scroll.options__1.label"
+        },
+        {
+          "value": "reduce-25",
+          "label": "t:sections.header.settings.reduce_logo_scroll.options__2.label"
+        },
+        {
+          "value": "reduce-50",
+          "label": "t:sections.header.settings.reduce_logo_scroll.options__3.label"
+        }
+      ],
+      "default": "reduce-0",
+      "label": "t:sections.header.settings.reduce_logo_scroll.label",
+      "info": "t:sections.header.settings.reduce_logo_scroll.info"
```

Comment:
Thanks for flagging this Ludo. I played around with it, and there are actually two settings here that only apply to logo images, so we should update both for clarity. I chatted with Olivia and we aligned on updating the setting labels:

- `Custom logo width` --> `Custom logo image width `
- `Reduce logo size on scroll` --> `Resize logo image on scroll` (Help text can remain as `Applies when sticky header is used.`) 
- Desktop logo position applies whether there is a logo image or not, so no change required here. 

Does that match your understanding?

Reply:
💯 Perfect 👌 thanks for looking into it Katy 🙂

Reply:
I tried to suggest the updates but I worried I did it wrong and deleted it 😨 Maybe I can ping you (or Lucas if he's feeling better) to pair for a few mins tomorrow. Would love to more confidently navigate these PRs and make the necessary updates rather than bugging you to do it!